doc/JSON_FLAGS.md

```diff
@@ -430,6 +430,12 @@ These branches are also the valid entries for the categories of `dreams` in `dre
 - ```INITIAL_PART``` When starting a new vehicle via the construction menu, this vehicle part will be the initial part of the vehicle (if the used item matches the item required for this part).
 - ```SCOOP``` Pulls items from underneath the vehicle to the cargo space of the part. Also mops up liquids. 
   - Uses the ```bonus``` tag to determine the maximum size of the item picked up
+- ```PLANTER``` Plants seeds into tilled dirt, spilling them when the terrain underneath is unsuitable. It is damaged by running it over non-```DIGGABLE``` surfaces.
+  - ```ADVANCED_PLANTER``` This planter doesn't spill seeds and avoids damaging itself on non-diggable surfaces.
+- ```REAPER``` Cuts down mature crops, depositing them on the square
+  - The ```bonus``` tag defines how productive the harvest can be.
+- ```PLOW``` Tills the soil underneath the part while active. Takes damage from unsuitable terrain at a level proportional to the speed of the vehicle.
+- ```EXTRA_DRAG``` tells the vehicle that the part exerts engine power reduction.
```

Comment:
`EXTRA_DRAG` is documented and even used in JSON, but there is not a single reference to it in C++. The code their still checks for the PLOW/REAPER etc. flags to add to `vehicle::extra_drag`.